packages/destination-actions/src/destinations/apolloio/index.ts

```diff
@@ -0,0 +1,61 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+import { IntegrationError, defaultValues } from '@segment/actions-core'
+
+import track from './track'
+export const baseURL = process?.env?.ACTIONS_APOLLOIO_BASE_URL_SECRET
+  ? `${process.env.ACTIONS_APOLLOIO_BASE_URL_SECRET}`
+  : 'https://apollo.io/'
+export const authURL = process?.env?.ACTIONS_APOLLOIO_AUTH_URL_SECRET
+  ? `${process.env.ACTIONS_APOLLOIO_AUTH_URL_SECRET}`
+  : 'https://apollo.io/'
```

Comment:
nit: This is a lot more succinct. [Null coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)
```suggestion
export const authURL = process?.env?.ACTIONS_APOLLOIO_AUTH_URL_SECRET ?? 'https://apollo.io/'
```